packages/otoggl/otoggl.0.3.1/opam

```diff
@@ -0,0 +1,46 @@
+opam-version: "2.0"
+synopsis: "Bindings for Toggl API in OCaml"
+description: "Bindings for Toggl API in OCaml"
+maintainer: ["Christophe Riolo Uusivaara"]
+authors: ["Christophe Riolo Uusivaara"]
+license: "MIT"
+homepage: "https://github.com/unitrack-time-tracking/OToggl"
+bug-reports: "https://github.com/unitrack-time-tracking/OToggl/issues"
+depends: [
+  "ocaml" {>= "4.06.0"}
+  "atdgen" {build & >= "2.2" & < "3"}
+  "atdgen-runtime" {>= "2" & < "3"}
+  "base64" {>= "3" & < "4"}
+  "containers" {>= "3" & < "4"}
```

Comment:
```suggestion
  "containers" {>= "3.6" & < "4"}
```
Are you sure that you are incompatible with a future version 4? I think you could relax this constraint, and we can add it back (if needed) when containers 4 is released and the revdep shows issues
```suggestion
  "containers" {>= "3.6"}
```

Same question for ppx_deriving and atdgen.

Reply:
The 3.6 bound is needed for
```
# (cd _build/default && /home/opam/.opam/4.10/bin/ocamlc.opt -w -40 -g -ppx '.ppx/b25831a27f6cc3ffc0d27a4be900dbfc/ppx.exe --as-ppx --cookie '\''library-name="Toggl"'\''' -bin-annot -I lib/.Toggl.objs/byte -I /home/opam/.opam/4.10/lib/angstrom -I /home/opam/.opam/4.10/lib/atdgen-runtime -I /home/opam/.opam/4.10/lib/base/caml -I /home/opam/.opam/4.10/lib/base64 -I /home/opam/.opam/4.10/lib/base64/rfc2045 -I /home/opam/.opam/4.10/lib/bigarray-compat -I /home/opam/.opam/4.10/lib/bigarray-overlap -I /home/opam/.opam/4.10/lib/bigarray-overlap/stubs -I /home/opam/.opam/4.10/lib/bigstringaf -I /home/opam/.opam/4.10/lib/biniou -I /home/opam/.opam/4.10/lib/bytes -I /home/opam/.opam/4.10/lib/containers -I /home/opam/.opam/4.10/lib/containers/monomorphic -I /home/opam/.opam/4.10/lib/easy-format -I /home/opam/.opam/4.10/lib/faraday -I /home/opam/.opam/4.10/lib/faraday-lwt -I /home/opam/.opam/4.10/lib/faraday-lwt-unix -I /home/opam/.opam/4.10/lib/fmt -I /home/opam/.opam/4.10/lib/gluten -I /home/opam/.opam/4.10/lib/gluten-lwt -I /home/opam/.opam/4.10/lib/gluten-lwt-unix -I /home/opam/.opam/4.10/lib/ke -I /home/opam/.opam/4.10/lib/logs -I /home/opam/.opam/4.10/lib/lwt -I /home/opam/.opam/4.10/lib/lwt/unix -I /home/opam/.opam/4.10/lib/lwt_ssl -I /home/opam/.opam/4.10/lib/magic-mime -I /home/opam/.opam/4.10/lib/mmap -I /home/opam/.opam/4.10/lib/mrmime/prettym -I /home/opam/.opam/4.10/lib/ocaml/threads -I /home/opam/.opam/4.10/lib/ocplib-endian -I /home/opam/.opam/4.10/lib/parsexp -I /home/opam/.opam/4.10/lib/pecu -I /home/opam/.opam/4.10/lib/piaf -I /home/opam/.opam/4.10/lib/piaf/h2 -I /home/opam/.opam/4.10/lib/piaf/h2-lwt -I /home/opam/.opam/4.10/lib/piaf/h2-lwt-unix -I /home/opam/.opam/4.10/lib/piaf/hpack -I /home/opam/.opam/4.10/lib/piaf/httpaf -I /home/opam/.opam/4.10/lib/piaf/httpaf-lwt -I /home/opam/.opam/4.10/lib/piaf/httpaf-lwt-unix -I /home/opam/.opam/4.10/lib/piaf/multipart -I /home/opam/.opam/4.10/lib/piaf/multipart_form -I /home/opam/.opam/4.10/lib/ppx_deriving/runtime -I /home/opam/.opam/4.10/lib/ppx_sexp_conv/runtime-lib -I /home/opam/.opam/4.10/lib/psq -I /home/opam/.opam/4.10/lib/ptime -I /home/opam/.opam/4.10/lib/ptime/os -I /home/opam/.opam/4.10/lib/re -I /home/opam/.opam/4.10/lib/re/posix -I /home/opam/.opam/4.10/lib/result -I /home/opam/.opam/4.10/lib/rresult -I /home/opam/.opam/4.10/lib/seq -I /home/opam/.opam/4.10/lib/sexplib -I /home/opam/.opam/4.10/lib/sexplib0 -I /home/opam/.opam/4.10/lib/ssl -I /home/opam/.opam/4.10/lib/stdlib-shims -I /home/opam/.opam/4.10/lib/stringext -I /home/opam/.opam/4.10/lib/uchar -I /home/opam/.opam/4.10/lib/unstrctrd -I /home/opam/.opam/4.10/lib/unstrctrd/parser -I /home/opam/.opam/4.10/lib/uri -I /home/opam/.opam/4.10/lib/uutf -I /home/opam/.opam/4.10/lib/yojson -no-alias-deps -open Toggl -o lib/.Toggl.objs/byte/toggl__Api.cmo -c -impl lib/api.ml)
# File "lib/api.ml", line 93, characters 8-16:
# 93 |         CCOption.(
#              ^^^^^^^^
# Error: Unbound module CCOption
```